launcher.py

```diff
@@ -36,6 +36,8 @@ async def main() -> None:
         dsn=core.CONFIG["DATABASE"]["dsn"]
     ) as pool, LogHandler(bot=bot) as handler:
         bot.logging_queue = asyncio.Queue()
+        bot.strip_after_prefix = True
+        bot.case_insensitive = True
```

Comment:
This change is present in #52. Where does it belong?

Reply:
Here. Deleted from #52